WcaOnRails/config/locales/en.yml

```diff
@@ -1168,7 +1168,7 @@ en:
       updated: "Updated registration"
       failed: "Could not update registration"
       deleted: "Successfully deleted your registration for %{comp}"
-      registered: "Successfully registered!"
+      registered: "Registration submitted - make sure to follow payment instructions to avoid disappointment!"
```

Comment:
What if a competition does not have payment instructions?

Reply:
Suggest we keep it as "payment instructions" as it is correct 99%+ of the time, and I think we save organizers more work by drawing attention to payment requirements for the average competition than if they need to follow up for other registration requirements for outlier competitions

Reply:
Can go back to the staff for feedback though

Reply:
@gregorbg wording has been updated based on staff feedback in "FEEDBACK REQUEST: Competition Notification Email Changes"